### PR TITLE
Activity stream update

### LINKED ISSUE #1224: Update Implementing Activity Feeds
https://documentation.platformos.com/developer-guide/activity-feeds/implementing-activity-feeds

- [x] verify that the topic describes latest best practice approach
- [x] extend if we have anything to add
- [ ] cross-link to community template repo as source code example (with short description)

### PR DESCRIPTION
**Pull request by hosszukalman**
Fixes #1224

### CODE REVIEW

File: app/views/pages/developer-guide/activity-feeds/activity-feeds.liquid [hunk 1]
```diff
@@ -10,6 +10,15 @@ An **activity** is a semantic description of an action.
 Get familiar with the [W3C Recommendation for Activity Streams 2.0](https://www.w3.org/TR/2017/REC-activitystreams-core-20170523/). Most examples and part of the implementation you'll find in this guide are based on this specification, that provides a JSON-based syntax that is sufficient to express metadata about activities in a rich, human-friendly but machine-processable and extensible manner. This can include constructing natural-language descriptions or visual representations about the activity, associating actionable information with various types of objects, communicating or recording activity logs, or delegation of potential actions to other applications.
 
 ## Activities
+
+Activities (activity list) is an activity stream and it is an immutable list of immutable objects (activities/events that happened it the past).
+
+- By design you can only add an entry
```
Comment (diana-lakatos) [inline app/views/pages/developer-guide/activity-feeds/activity-feeds.liquid]: If this means that only one entry can be added, we should write it as "By design, you can only add one entry".

**Review by diana-lakatos — APPROVED** [review]
@hosszukalman I reviewed and edited, and also left one comment. Thank you very much! 
@godot Please review! Thank you!

File: app/views/pages/developer-guide/activity-feeds/activity-feeds.liquid (hunk 1 above)
Comment (hosszukalman) [reply app/views/pages/developer-guide/activity-feeds/activity-feeds.liquid]: It means that by design an entry can be added only, update and delete operands are denied. What do you think, do I need to modify this line?

File: app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+#### 2: Get them when you load the main activities
+
+```liquid
+{% liquid
+  assign current_page = context.params.current_page | to_positive_integer: 1
+
+  graphql g = 'get_activities', user_id: context.current_user.id, feed_type: 'UserNotifications'
+  graphql gclicked = 'get_activities', user_id: context.current_user.id feed_type: 'UserNotifications.Click'
```
Comment (godot) [inline app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid]:
```suggestion
  graphql gclicked = 'get_activities', user_id: context.current_user.id, feed_type: 'UserNotifications.Read'
```

File: app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+
+#### 4: Mark all notifications as clicked.
+
+```liquid
+{% liquid 
+  assign recipients = context.current_user.id | split: "|"
+  graphql notifications = 'get_activities', user_id: context.current_user.id | dig: 'streams', 'results'
+
+  for notification in notifications
+    graphql pg = 'activity_publish',  uuid: notification.uuid, feed_ids: recipients, feed_type: 'UserNotifications.Click'
```
Comment (godot) [inline app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid]:
```suggestion
    graphql pg = 'activity_publish',  uuid: notification.uuid, feed_ids: recipients, feed_type: 'UserNotifications.Read'
```

File: app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+- but you want to keep the activites that the user has not clicked yet unread 
+
+You can implement this scenario with the following steps:
+
+#### 1: Create another stream to store the clicked/read activities
+
+```liquid
+{% raw %}
+{% assign recipients = context.current_user.id | split: "|" %}
+{% graphql gp = 'activity_publish', uuid: activity.uuid, feed_ids: recipients, feed_type: 'UserNotifications.Clicked' %}
```
Comment (godot) [inline app/views/pages/developer-guide/activity-feeds/handling-read-activites.liquid]:
```suggestion
{% graphql gp = 'activity_publish', uuid: activity.uuid, feed_ids: recipients, feed_type: 'UserNotifications.Read' %}
```

**Review by godot — CHANGES_REQUESTED** [review]

**Review by godot — APPROVED** [review]

**Pull request merged**